prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -87,7 +87,7 @@ fi
 [[ $SHMTHROW == 0 ]] && ARGS_ALL+=" --shm-throw-bad-alloc 0"
 [[ ! -z $SHM_MANAGER_SHMID ]] && ARGS_ALL+=" --shm-no-cleanup on --shmid $SHM_MANAGER_SHMID"
 [[ $NORATELOG == 1 ]] && ARGS_ALL+=" --fairmq-rate-logging 0"
-if [[ $EPNSYNCMODE == 1 ]] || [[ `which numactl 2> /dev/null | wc -l` != "0" && `numactl -H | grep "node . size" | wc -l` -ge 2 ]]; then
+if [[ $EPNSYNCMODE == 1 ]] || type numactl >/dev/null 2>&1 && [[ `numactl -H | grep "node . size" | wc -l` -ge 2 ]]; then
```

Comment:
No output should go to stdout (It regularly ends up in JSON and XML files), why do you have 2>&1? Can't we just remove that?

Reply:
Because BSD `type` / `which` like the one on macOS prints "numactl not found" on stdout. I think the only thing you can rely on is that when the argument does not exist they will exit with a non zero exit code.

Reply:
Also notice that here I do `>/dev/null 2>&1` which means both go to `/dev/null` (the logic is: first you redirect stdout to dev null and then you redirect stderr to the **redirected** stdout).

Reply:
Ahh, ok, didn't know that. I assumed it would send stdout to /dev/null and afterwards make stderr write to stdout. Wasn't aware that the `> /dev/null` also applied to that. Usually, to send everything to `/dev/null` I use `&> /dev/null`. :)

Reply:
Does `&>` actually work in bash? I thought it was zsh only and `>/dev/null 2>&1` is for sure the "POSIX" way. Anyways, given we have other bash-isms I am also fine if you want to change to `&>`.

Reply:
`&>` for sure works in bash.
What I find a bit confusing is that for a pipe one needs `|&` ...
Anyway, I already merged it anyway.